test/fixtures/core.scale/crossAlignment/cross-align-right-far-clipped.js

```diff
@@ -0,0 +1,32 @@
+module.exports = {
+	config: {
+		type: 'bar',
+		data: {
+			datasets: [{
+				data: [1, 2, 3],
+			}],
+			labels: ['Long long long long label 1', 'Label 2', 'Less more longer label 3']
+		},
+		options: {
+			indexAxis: 'y',
+			scales: {
+				y: {
+					position: 'right',
+					ticks: {
+						crossAlign: 'far',
+					},
+					afterFit: axis => {
+						axis.width = 64;
+					},
+				},
+			}
+		}
+	},
+	options: {
+		spriteText: true,
+		canvas: {
+			height: 256,
+			width: 512
+		}
+	}
+};
```

Comment:
Adding `tolerance: 0.01`  might help it pass in both browsers.

Reply:
@iamarcel are you able to try this out? Would love to get this merged

Reply:
That will fix it

Reply:
Thanks for continuing this while I was out and half forgetting about this! :)